packages/kbn-apm-synthtrace-client/src/lib/logs/index.ts

```diff
@@ -63,6 +63,7 @@ export type LogDocument = Fields &
     'event.duration': number;
     'event.start': Date;
     'event.end': Date;
+    test_field: string | string[];
```

Comment:
What is the use of this new property?

Reply:
This has been mostly added for tests. I will try to explain

Q. Why do i need this additional field ?
A. Because i needed a field which is both string and string[]

Q. Why can't you extend an existing field which is already string to also be string[]
A. I tried, the problem is most of these fields are APM Fields or some other fields and i don't want to impact them

Q. OK OK, but why do you need a field with string and string[]?
A. I need to generate a scenario, where a field can have something like this

```test_field = [1024_CHAR, 'hello:world', ANOTHER_1024_CHAR]```

In such situations, only the array[0] and array[2] should be ignored values and not array[1]

I hope this explains

Reply:
> In such situations, only the array[0] and array[2] should be ignored values and not array[1] as it has not crossed 1024 length

In that case do we get the same field indexed and ignored at the same time?

Reply:
yup